src/h3/api/basic_int/__init__.py

```diff
@@ -996,3 +996,70 @@ def great_circle_distance(latlng1, latlng2, unit='km'):
         lat2, lng2,
         unit = unit
     )
+
+
+def cell_to_vertex(h, vertex_num):
+    """
+    Return a (specified) vertex of an H3 cell.
+
+    Parameters
+    ----------
+    h : H3Cell
+    vertex_num : int
+        Vertex number (0-5)
+
+    Returns
+    -------
+    The vertex
+    """
+    h = _in_scalar(h)
+    return _cy.cell_to_vertex(h, vertex_num)
+
+
+def cell_to_vertexes(h):
+    """
+    Return a list of vertexes of an H3 cell. 
+    The list will be of length 5 for pentagons and 6 for hexagons.
+
+    Parameters
+    ----------
+    h : H3Cell
+
+    Returns
+    -------
+    A list of vertexes
+    """
+    h = _in_scalar(h)
+    mv = _cy.cell_to_vertexes(h)
+    arr = [str_to_int(a) for a in _out_collection(mv)]
```

Comment:
Thanks for making adjustments, but this will still return `int`s for each API. We want the output to depend on the API:

- list of `int` for `h3.api.basic_int`
- list of `str` for `h3.api.basic_str`
- `numpy.array` for `h3.api.numpy_int`
- `memoryview` of `int` for `h3.api.memview_int`

To do that, the last operation should be `_out_collcation(mv)`. The `_remove_zeros()` function should help with this: https://github.com/uber/h3-py/blob/master/src/h3/_cy/memory.pyx#L113

Reply:
Is this line still needed? It looks as if you can `return _out_collection(mv)`?